docs/config/prompt.md

```diff
@@ -94,6 +94,7 @@ SPACESHIP_PROMPT_ORDER=(
   exit_code      # Exit code section
   sudo           # Sudo indicator
   char           # Prompt character
+  gleam          # Gleam section
```

Comment:
This should not be after `char`. It should be along with other language/runtime sections.